pom.xml

```diff
@@ -237,20 +237,16 @@
                 <version>${carbon.identity.framework.version}</version>
             </dependency>
             <dependency>
-                <groupId>org.wso2.carbon.identity.organization.management</groupId>
+                <groupId>org.wso2.carbon.identity.organization.management.core</groupId>
                 <artifactId>org.wso2.carbon.identity.organization.management.service</artifactId>
-                <version>${carbon.identity.organization.management.version}</version>
-                <exclusions>
-                    <exclusion>
```

Comment:
looks like following comes from the framework dependency which will cause cyclic dependency, we will have to upgrade the identity framework version or exclude org management from the dep.
```
\- org.wso2.carbon.identity.organization.management:org.wso2.carbon.identity.organization.management.service:jar:1.0.65:compile
     \- org.wso2.carbon.identity.organization.management:org.wso2.carbon.identity.organization.management.authz.service:jar:1.1.5:compile
        \- org.wso2.carbon.identity.auth.rest:org.wso2.carbon.identity.authz.service:jar:1.6.2-SNAPSHOT:compile
           \- org.wso2.carbon.identity.auth.rest:org.wso2.carbon.identity.auth.service:jar:1.6.2-SNAPSHOT:compile
```

Reply:
upgraded framework version